src/App.jsx

```diff
@@ -76,6 +76,7 @@ const Block = ({ index, onChange, onClick, children }) => {
   };
 
   const handleOnClick = e => {
+    //What does this function do?
     e.stopPropagation();
```

Comment:
it stops events from bubbling up to the parent. Events in JS keep on "bubbling" up until they either hit a stopPropagation() or usually the <html> tag

Reply:
the html tag* I wrote it with brackets and github deleted it lol